app/scripts/constants.js

```diff
@@ -5,6 +5,9 @@
 // Reference: https://docs.openshift.org/latest/install_config/web_console_customization.html#loading-custom-scripts-and-stylesheets
 
 // NOTE: Update extensions/examples/online-extensions.js if you add a new help link to this map.
+var gettext = function(s) {
```

Comment:
I assume this is here so the gettext compiler finds theses strings since constants.js lives outside of the angular scope.  Can you add comments here explaining that so that someone doesn't try to remove it later thinking its just a no-op